repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/fullWidthImage.ftl

```diff
@@ -0,0 +1,5 @@
+<#ftl output_format="HTML">
+
+<#macro fullWidthImageSection section>
+    <h1>Full Width Image goes here</h1>
```

Comment:
Could this placeholder message end up being shown on production?

Reply:
@GeoffHayward Yes, but only if the section is invoked. It's a good point though. Luckily I have the Front End PR ready to go as soon as this is merged in, so we should be ok.